buffer/src/main/java/io/netty5/buffer/api/bytebuffer/NioBuffer.java

```diff
@@ -47,7 +47,6 @@
 import static io.netty5.buffer.api.internal.Statics.MAX_BUFFER_SIZE;
 import static io.netty5.buffer.api.internal.Statics.bbput;
 import static io.netty5.buffer.api.internal.Statics.bbslice;
-import static io.netty5.buffer.api.internal.Statics.bufferIsClosed;
```

Comment:
Thanks for cleaning the unused imports up! Out of curiosity, how did our `Checkstyle` rules not fail the build on this?